android_p/google_diff/cel_apl/system/bt/0004-Fix-for-Play-pause-operation-for-DUT.patch

```diff
@@ -0,0 +1,95 @@
+From 4c659464e0ff6793807d39a345e8061453bc6983 Mon Sep 17 00:00:00 2001
+From: Nitin Rawat <nitin.rawat@intel.com>
+Date: Fri, 22 Feb 2019 13:23:32 +0530
+Subject: [PATCH 4/4] Fix for Play/pause operation for DUT
+
+Play pause operation was not working from DUT as reference
+device was not sending playbackstatus changed notification.
+Added code to handle this at DUT side itself using
+getplaystatus response.
+
+Change-Id: If1d4af2e725efc6993de3a9482104992791a65f5
+Tracked-On: OAM-72645
+Signed-off-by: Nitin Rawat <nitin.rawat@intel.com>
+---
+ btif/src/btif_rc.cc        | 26 ++++++++++++++++++++++++--
+ stack/avrc/avrc_pars_ct.cc |  3 ++-
+ 2 files changed, 26 insertions(+), 3 deletions(-)
+
+diff --git a/btif/src/btif_rc.cc b/btif/src/btif_rc.cc
+index c226b5f..93be71c 100644
+--- a/btif/src/btif_rc.cc
++++ b/btif/src/btif_rc.cc
+@@ -347,7 +347,10 @@ static bool absolute_volume_disabled(void);
+ static rc_cb_t btif_rc_cb;
+ static btrc_callbacks_t* bt_rc_callbacks = NULL;
+ static btrc_ctrl_callbacks_t* bt_rc_ctrl_callbacks = NULL;
+-
++//These 2 flag has been introduced to avoid playback status changed handler
++//getting called again and again even if the status has not changed
++static int play_flag=0;
++static int pause_flag=0;
+ /*****************************************************************************
+  *  Static functions
+  *****************************************************************************/
+@@ -3204,6 +3207,7 @@ static void handle_notification_response(tBTA_AV_META_MSG* pmeta_msg,
+          * if the play state is playing.
+          */
+         if (p_rsp->param.play_status == AVRC_PLAYSTATE_PLAYING ||
++	    p_rsp->param.play_status == AVRC_PLAYSTATE_PAUSED  ||
```

Comment:
-Follow the same indentation as other lines
-Remove extra space between PAUSED and ||